site/guide/11-conclusion.md

```diff
@@ -8,12 +8,12 @@ we're happy to take the best ideas. If you have something in mind, please
 ## Getting Help
 
 If you find yourself having trouble developing Rocket applications, you can get
-help via the `#rocket` IRC channel on the [Mozilla IRC
-Server](https://wiki.mozilla.org/IRC) at `irc.mozilla.org` and the bridged
+help via the `#rocket` IRC channel on the [freenode IRC
+server](https://freenode.net/) at `irc.freenode.net` and the bridged
 [Rocket room on Matrix](https://riot.im/app/#/room/#mozilla_#rocket:matrix.org).
```

Comment:
The bridge link should be updated, too - we now have `#freenode_#rocket:matrix.org` (automatically bridged) and `#rocket:mozilla.org` (manually added by @SergioBenitez ).